src/components/MultiHopTrade/components/MultiHopTradeConfirm/components/FeeStep.tsx

```diff
@@ -29,32 +26,14 @@ export const FeeStep = ({ isLastStep }: FeeStepProps) => {
   } = useLocaleFormatter()
   const translate = useTranslate()
   const [showFeeModal, setShowFeeModal] = useState(false)
-  const amountAfterDiscountUserCurrency = useAppSelector(selectQuoteAffiliateFeeUserCurrency)
-  const amountBeforeDiscountUserCurrency = useAppSelector(
-    selectPotentialAffiliateFeeAmountUserCurrency,
-  )
-  const potentialAffiliateBps = useAppSelector(selectActiveQuotePotentialAffiliateBps)
+
+  // use the fee data from the actual quote in case it varies from the theoretical calculation
```

Comment:
Ditto question WRT free trades

Reply:
solved in 4c28a00c7941ac2e2c4d2320b442c6a89ee3d7fa